src/Storage/Storage.php

```diff
@@ -21,6 +21,8 @@ class Storage
 
     const DEVICE_LINODE = 'linode';
 
+    const DEVICE_ORACLEOBJECT = 'oracleobject';
```

Comment:
This should be `DEVICE_ORACLE_OBJECT` just like `DEVICE_DO_SPACES`:

```suggestion
    const DEVICE_ORACLE_OBJECT = 'oracleobject';
```